utils/model_uploader/model_autotracing.py

```diff
@@ -0,0 +1,551 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# This program is run by "Model Auto-tracing & Uploading" workflow
+# (See model_uploader.yml) to perform model auto-tracing and prepare
+# files for uploading to OpenSearch model hub.
+
+import argparse
+import os
+import shutil
+import sys
+import warnings
+from typing import List, Optional, Tuple
+from zipfile import ZipFile
+
+import numpy as np
+from mdutils.fileutils import MarkDownFile
+from numpy.typing import DTypeLike
+from sentence_transformers import SentenceTransformer
+
+THIS_DIR = os.path.dirname(__file__)
+ROOT_DIR = os.path.join(THIS_DIR, "../..")
+sys.path.append(ROOT_DIR)  # Required for importing OPENSEARCH_TEST_CLIENT
```

Comment:
We should add some comment to describe why do we need to do this as this is different from other files.